pkg/antctl/raw/multicluster/common/common.go

```diff
@@ -213,6 +213,21 @@ func deleteServiceAccounts(cmd *cobra.Command, k8sClient client.Client, namespac
 	}
 }
 
+func GetTokenSecret(secret corev1.Secret) corev1.Secret {
```

Comment:
Maybe name it sth. like: ConvertMemberTokenSecret(). Also good to add some comments saying the result Secret struct is for generating the token Secret manifest for join.

Reply:
And could you change CreateMemberToken() to use this func as well?

Reply:
@jianjuns it is already used in CreateMemberToken()

Reply:
Ok.